LowLevelDesign/tic-tac-toe/src/Board.ts

```
@@ -0,0 +1,48 @@
+import { Cell } from "./Cell";
+import { Symbol } from "./enum/Symbol";
```

Biome (2.1.2)

[error] Do not shadow the global "Symbol" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)